docs/reference/autoscaling/autoscaling-designed-for-note.asciidoc

```diff
@@ -0,0 +1,2 @@
+[NOTE]
+Autoscaling is designed for indirect use by ECE/ESS and ECK. Direct use is not supported.
```

Comment:
I added the links in b8c73509249268858134b21484553dee465e7738, but kept `indirect` rather than `internal`, since ESS/ECE/ECK users will be able to configure the decider settings indirectly, using the setting names documented here through those products.